tests/drivers/flash_simulator/prj.conf

```diff
@@ -3,3 +3,4 @@ CONFIG_FLASH=y
 CONFIG_FLASH_SIMULATOR=y
 CONFIG_FLASH_SIMULATOR_DOUBLE_WRITES=n
 CONFIG_FLASH_SIMULATOR_ERASE_PROTECT=y
+CONFIG_FLASH_SIMULATOR_UNALIGNED_READ=n
```

Comment:
This change is not reflected in the commit message.

Reply:
(Would be nice to justify why the test requires aligned access)

Reply:
This is just because test checks this part and I do not wish to implement 2 different tests (repeating the whole test with just this parts disabled does not add goods).
I am not sure if we should explain everything in the PR message? PR message should be short not coming into details? Anyone? Should I really describe in PR message that I have added this flag and why?

Reply:
The introduction of this Kconfig flag is well-described in the commit message. Add a sentence explaining why the tests needs to run with the Kconfig symbol undefined.

Reply:
A note added.